core/base/src/main/java/alluxio/exception/NameTooLongException.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.exception;
+
+import alluxio.AlluxioURI;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * The exception thrown when a file already exists in Alluxio.
+ */
+@ThreadSafe
+public class NameTooLongException extends AlluxioException {
```

Comment:
We are in the mode of deprecating exceptions to consolidate on this set of exceptions https://github.com/Alluxio/alluxio/tree/dfbb2406a6f3e03b93fd6da7ce46fd81dcacdd97/core/base/src/main/java/alluxio/exception/status Let's not introduce another new exception

Reply:
have deleted NameTooLongException.